phoenix-core/src/main/java/org/apache/hadoop/hbase/regionserver/DataTableLocalIndexRegionScanner.java

```diff
@@ -91,15 +92,19 @@ public boolean next(List<Cell> outResult, ScannerContext scannerContext) throws
     public boolean next(List<Cell> results) throws IOException {
         List<Cell> dataTableResults = new ArrayList<Cell>();
         boolean next = super.next(dataTableResults);
-        addMutations(dataTableResults);
+        try {
+            addMutations(dataTableResults);
+        } catch (SQLException e) {
+            e.printStackTrace();
```

Comment:
We shouldn't ever be swallowing an exception and printing to stdout/stderr. We should log it at worst. Any reason this exception shouldn't bubble up?